app/models/api_version.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class APIVersion
```

Comment:
1. Would keep dynamically generated methods since if we add v2 it will automatically have a method
2. Also want to keep it as standard ActiveModel interface to treat it as any other model